src/ui_basic/table.cc

```diff
@@ -145,7 +147,20 @@ void Table<void*>::set_column_tooltip(uint8_t col, const std::string& text) {
 	assert(col < columns_.size());
 	Column& column = columns_.at(col);
 	assert(column.btn);
-	column.btn->set_tooltip(text);
+	column.user_tooltip = text;
+	column.update_tooltip(col == get_sort_column());
+}
+
+void Table<void*>::Column::update_tooltip(bool sorted) {
+	const std::string click_tooltip =
+	   sorted ? _("Click to reverse sorting") : _("Click to sort by this column");
+	if (user_tooltip.empty()) {
+		btn->set_tooltip(click_tooltip);
+	} else {
+		/** TRANSLATORS: Tooltip format for table headers: %1 is column description, %2 is
+		    "Click to sort ..." or "Click to reverse..." */
+		btn->set_tooltip(format(_("%1s<br>(%2s)"), user_tooltip, click_tooltip));
```

Comment:
Shouldn't that be:
```suggestion
		btn->set_tooltip(format(_("%1$s<br>(%2$s)"), user_tooltip, click_tooltip));
```

Reply:
Oops. Thank you. :) And oops again, I forgot it was a suggestion.